contracts/contracts/gateway/router/CheckpointingFacet.sol

```diff
@@ -43,6 +45,35 @@ contract CheckpointingFacet is GatewayActorModifiers {
         execBottomUpMsgs(checkpoint.msgs, subnet);
     }
 
+    /// @notice creates a new bottom-up checkpoint with activity report
```

Comment:
It's because when we were discussing the commitment generation, we were talking about calling into fvm actor from solidity contract, so if we followed this implementation, we will update the `createBottomUpCheckpoint` method to call fvm actors. But then we also implemented the `createBottomUpCheckpointWithActivities` method, basically fendermint derives the commiment. The first approach was pushed to later phases, this method remained like a `TODO`. But we can remove it for now.